### ISSUE TITLE
Improve Daje Docs

### ISSUE THREAD

**Issue by davideimola** [Schroedinger-Hat/Daje#7]

**Comment by lodenrogue** [comment]
I agree. Stumbling across this project for the first time I'm not sure how to use it

**Comment by davideimola** [comment]
Hi @lodenrogue, currently the code in the main branch is kind of useless, because we are trying to recreate the project as a CLI.

If you want, you can use the old project by forking this release https://github.com/Schrodinger-Hat/Daje/releases/tag/v0.1.0
